bin/travis_scripts/check_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
What is the point to use set -e if there is only 1 command?

Reply:
Reserved for future usage :)

Reply:
Fixed